core/arch/arm/mm/tee_mm.c

```diff
@@ -139,13 +141,16 @@ tee_mm_entry_t *tee_mm_alloc(tee_mm_pool_t *pool, size_t size)
 {
 	size_t psize;
 	tee_mm_entry_t *entry;
-	tee_mm_entry_t *nn;
+	tee_mm_entry_t *nn = NULL;
 	size_t remaining;
+	uint32_t exceptions;
 
 	/* Check that pool is initialized */
 	if (!pool || !pool->entry)
 		return NULL;
 
+	exceptions = cpu_spin_lock_xsave(&pool->lock);
+
```

Comment:
minor: there is a comment below that could be removed (`/* Protect with mutex (multi thread) */`)